src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    Ball getRandomBall() {
+        int count = 100;
+        Random number = new Random();
+        ColorSupplier color = new ColorSupplier();
```

Comment:
```suggestion
    private final Random random = new Random();
    private final ColorSupplier colorSupplier = new ColorSupplier();
    
    public Ball getRandomBall() {
        int count = 100;
```
it's a class-level variables.